streaming/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@mastodon/streaming-server",
+  "license": "AGPL-3.0-or-later",
+  "description": "Mastodon's Streaming Server",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mastodon/mastodon.git"
+  },
+  "scripts": {
+    "start": "node index.js"
+  },
+  "main": "index.js",
+  "dependencies": {
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "jsdom": "^21.1.1",
+    "npmlog": "^7.0.1",
+    "pg": "^8.5.0",
+    "pg-connection-string": "^2.5.0",
```

Comment:
One of the connection dependencies received an update, looks like I just need to regenerate the lockfile